docs/Parameter.md

```diff
@@ -0,0 +1,82 @@
+---
+file_format: mystnb
+kernelspec:
+  name: python3
+mystnb:
+  number_source_lines: true
+---
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+%config InlineBackend.figure_formats = ['svg']
+```
+
+# Parameter Space
+
+The `mqt.bench.get_benchmark` method has the following signature:
+
+```{eval-rst}
+.. automodule:: mqt.bench.benchmark_generation
+    :no-index:
+    :members: get_benchmark
+```
+
+- `benchmark_name` (see {doc}`details <Benchmark_selection>`) : `"ae"`, `"bv"`, `"dj"`, `"grover-noancilla"`, `"grover-v-chain"`, `"ghz"`, `"graphstate"`,
+  `"qaoa"`, `"qft"`, `"qftentangled"`, `"qnn"`, `"qpeexact"`, `"qpeinexact"`,
+  `"qwalk-noancilla"`, `"qwalk-v-chain"`, `"randomcircuit"`, `"vqerealamprandom"`, `"vqesu2random"`, `"vqetwolocalrandom"`,
+  `"wstate"`, `"shor"`
+- `level`: `0` or `"alg"`, `1` or `"indep"`, `2` or `"nativegates"`, `3` or `"mapped"`
+- `circuit_size`: for most of the cases this is equal to number of qubits
+  (all scalable benchmarks except `"qwalk-v-chain"` and `"grover-v-chain"`) while for all other the qubit number is higher
+- `compiler_settings`: Optimization level for `"qiskit"` (`0`-`3`), exemplary shown:
+
+```python
+from mqt.bench import CompilerSettings, QiskitSettings
+
+compiler_settings = CompilerSettings(
+    qiskit=QiskitSettings(optimization_level=1),
+)
+```
+
+with:
+
+```{eval-rst}
+.. automodule:: mqt.bench.benchmark_generation
+    :no-index:
+    :members: CompilerSettings, QiskitSettings
+```
+
+- `gateset`: `"ibm_falcon"`, `"ibm_heron_r1"`, `"rigetti"`, `"ionq"`, `"iqm"`, `"oqc"`, or `"quantinuum"` (required for "nativegates" level)
+- `device_name`: `"ibm_washington"`, `"ibm_montreal"`, `"rigetti_aspen_m3"`, `"ionq_harmony"`, `"ionq_aria1"`, `"oqc_lucy"`, or `"quantinuum_h2"` (required for "mapped" level)
+
+## Quantum Circuit Compiler Support
```

Comment:
I suggest to remove this paragraph since we do not plan to extend the number of supported compilers in the near future.